iOS/Flows/People/PeopleCoordinator+Extensions.swift

```diff
@@ -144,10 +136,20 @@ extension PeopleCoordinator: MFMessageComposeViewControllerDelegate {
             }
         case .sent:
             controller.dismiss(animated: true) {
-                if let contact = self.selectedContact {
-                    self.showSentAlert(for: contact)
-                    self.updateInvitation()
+                if let phone = controller.recipients?.first, let person = self.peopleToInvite.first(where: { person in
```

Comment:
I added this so we didn't have to store a local var, and could more accurately associate who was texted.